.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,73 @@
+name: 🐞 Bug
+description: File a bug/issue
```

Comment:
Here is also good to keep the existing name.
File a bug/issue -> Create a report to help us improve